_config.yml

```diff
@@ -370,3 +370,7 @@ authors:
     name: Jaakko Saha
     email: 6a19b5eb0335707f33dfeb263649da77
     url: https://www.linkedin.com/in/jaakko-saha-490255107/
+  sergei.vorobyov:
+    name: Sergei Vorobyov
+    email: e196ababec915b5e5100983d30f84253
+    url: https://example.com
```

Comment:
Here could have some web presence of yours or `https://www.solita.fi` if you prefer to not adding a personal site. Still better to have a working targer.